src/views/AddItem.css

```diff
@@ -0,0 +1,89 @@
+.add-item {
+	display: flex;
+	flex-direction: column;
+	margin: 0 auto;
+}
+
+.add-item-header {
+	text-align: center;
+}
+
+.add-item-form {
+	border: 2px solid #b65c1a;
+	padding: 15px;
+	border-radius: 5px;
+	background-color: #ffffff;
+	margin-top: 25px;
+	filter: drop-shadow(0px 4px 4px rgba(0, 0, 0, 0.25));
+}
+
+.form-group {
+	padding: 0px 0px 15px 0px;
+	margin: 0 15px;
+}
+
+.item-input {
+	border: 2px solid #c18c5d !important;
+	border-radius: 9px !important;
+}
+
+#purchase-date-label {
+	margin-bottom: 15px;
+}
+
+.form-group-button {
+	text-align: center;
+	margin-bottom: 15px;
+}
+
+.purchase-date-row {
+	margin-top: 20px;
+	margin-bottom: 20px;
+	padding: 10px;
+	border-radius: 9px;
+}
+
+.radio-button {
+	margin: 10px 0px 0px 0px !important;
```

Comment:
For paddings and margins, since we're using Bootstrap, another way to do this is to use Bootstrap's utility classes. For example, if you wanted to add some padding to an element like a button it would be something like `className="p-2"`. Or for margin-top it would be, `className="mt-2"`. More spacing info here: https://getbootstrap.com/docs/5.0/utilities/spacing/

You did a great job overall, though! CSS can be tricky.

Reply:
ohhh this is super helpful! I'm going to look into it--the radio buttons in particular were a little challenging to get positioned right, but I'm hoping this will help!

Reply:
Another update on this! This was super helpful feedback, and I moved a ton from the css file back into the class names. For the radio button margins, I ended up leaving it in the CSS file because it was super challenging to get the inputs centered in the div. Open to any feedback others have for how to adjust that!

Reply:
Great suggestion, @arcan9! Yay teamwork!